dev/drivers/scripts/stats/cam/jevs_cam_rrfs_snowfall_stats.sh

```diff
@@ -0,0 +1,56 @@
+#PBS -N jevs_cam_rrfs_snowfall_stats
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=02:30:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1:mem=256GB
+#PBS -l debug=true
+
+set -x
+export model=evs
+export machine=WCOSS2
+
+# ECF Settings
+export SENDMAIL=YES
+export SENDECF=YES
+export SENDCOM=YES
+export KEEPDATA=YES
+export SENDDBN=NO
+export SENDDBN_NTC=
+export job=${PBS_JOBNAME:-jevs_cam_rrfs_snowfall_stats}
+export jobid=$job.${PBS_JOBID:-$$}
+export SITE=$(cat /etc/cluster_name)
+export USE_CFP=YES
+export nproc=128
+
+# General Verification Settings
+export NET="evs"
+export STEP="stats"
+export COMPONENT="cam"
+export RUN="atmos"
+export VERIF_CASE="snowfall"
+export MODELNAME="rrfs"
+
+# EVS Settings
+export HOMEevs="/lfs/h2/emc/vpppg/noscrub/$USER/EVS"
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${evs_ver}}
+export config=$HOMEevs/parm/evs_config/cam/config.evs.prod.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.${MODELNAME}
+
+# Load Modules
+source $HOMEevs/versions/run.ver
+module reset
+module load prod_envir/${prod_envir_ver}
+source $HOMEevs/dev/modulefiles/$COMPONENT/${COMPONENT}_${STEP}.sh
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
+
+# Developer Settings
+export envir=prod
+export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
+export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d/$STEP/$COMPONENT
+export vhr=${vhr:-${vhr}}
+export MAILTO="marcel.caron@noaa.gov,alicia.bentley@noaa.gov"
+
+# Job Settings and Run
+. ${HOMEevs}/jobs/JEVS_CAM_STATS
```

Comment:
Some of the rrfs stats drivers end here, others have a short summary, that looks like this:
```
############################################################
# Purpose: This job will generate the grid2obs statistics 
#          for the RRFS_FIREWXNEST model and generate stat 
#          files.
############################################################
exit
```
Is that summary and exit required for EVS or EE2? Or is it OK to end w/ the call to the J-job?

The drivers that end abruptly include:
```
jevs_cam_rrfs_grid2obs_stats.sh
jevs_cam_rrfs_precip_stats.sh
jevs_cam_rrfs_snowfall_stats.sh
```

Reply:
I'm not aware of either the summary or exit being EVS or EE2 standards, but the summary does seem to be the norm across most components.  I'll add the summaries and but not the "exit".